ansible_base/lib/utils/views/django_app_api.py

```diff
@@ -29,4 +33,6 @@
 
 
 class AnsibleBaseDjangoAppApiView(parent_view_class):
-    pass
+    # In case an app is using features like authentication, resources, or RBAC
+    # but not using rest_filters app, we need to specify the same filter backends
+    filter_backends = (TypeFilterBackend, FieldLookupBackend, SearchFilter, OrderByBackend)
```

Comment:
https://github.com/ansible/django-ansible-base/pull/212 has re-done this, and we should use that code. It will cause a conflict, and it should win. The UI needed it, but I guess I might be able to revert it for now. I'm a bit torn, can you look at the other PR and give your preferred plan of action?